exercises/beer-song/beer_test.go

```diff
@@ -49,6 +57,7 @@ var verseTestCases = []struct {
 func TestBottlesVerse(t *testing.T) {
 	for _, tt := range verseTestCases {
 		actualVerse, err := Verse(tt.verse)
+		var _ error = err
 		if actualVerse != tt.expectedVerse {
```

Comment:
should this only be checked `if !tt.expectErr`? that would be in line with the "Ignore the normal return value when an error expected"